invenio_app_ils/ui/src/invenio_app_ils/common/helpers.js

```diff
@@ -0,0 +1,8 @@
+export const shortenString = input => {
```

Comment:
I think [lodash.truncate](https://lodash.com/docs/4.17.11#truncate) does the same thing.